src/odemis/acq/stream/_sync.py

```diff
@@ -1923,38 +1967,29 @@ def _runAcquisition(self, future):
             # initialize leeches
             leech_np, leech_time_ppx = self._startLeeches(px_time, tot_num, (rep[1], rep[0]))
 
-            self._im_needs_recompute = threading.Event()
-            self._live_update_period = 0.33 # s
-            self._liveImageThread = threading.Thread(target = self._image_thread,
-                                                     args = (weakref.ref(self), self._live_update_period),
-                                                     name = "Live update of the acquisition")
-            self._liveImageThread.daemon = True
-            self._liveImageThread.start()
-
             # number of spots scanned so far
             spots_sum = 0
             while spots_sum < tot_num:
-                px_idx = (spots_sum//rep[0], spots_sum%rep[0]) #current pixel index
                 # Acquire the maximum amount of pixels until next leech
-                self._npixels2scan = min([np for np in leech_np if np is not None] +
+                npixels2scan = min([np for np in leech_np if np is not None] +
                                          [tot_num - spots_sum] + [int(self._live_update_period // px_time + 1)]) # max, in case of no leech
-                #TODO K.K. update unit tests to test for:
+                #TODO update unit tests to test for:
                 # cornercase where leech is set far after update_interval and therefore a non rectangular block may be scanned
                 # case where leech and update interval run semi in phase/out of phase
-                n_y, n_x = leech.get_next_rectangle((rep[1], rep[0]), spots_sum, self._npixels2scan)
-                self._npixels2scan = n_x * n_y
+                n_y, n_x = leech.get_next_rectangle((rep[1], rep[0]), spots_sum, npixels2scan) #method to only scan
+                # rectangular blocks of pixels which start at the beginning of a row
```

Comment:
Either put the long comment in a separate line *before* the function call, or make the comment fit on the line. Maybe you can just write `# also ensure it's a rectangle`